src/analyses/variable-sensitivity/abstract_enviroment.h

```diff
@@ -77,13 +77,13 @@ class abstract_environmentt
 
   abstract_object_statisticst gather_statistics(const namespacet &ns) const;
 
-protected:
-  bool bottom;
-
   // We may need to break out more of these cases into these
   virtual abstract_object_pointert eval_expression(
```

Comment:
:question: (will check commit message) why does this need to become unprotected?

Reply:
:no_entry_sign: Commit message does not explain why - could you pull this into own commit and justify why the API is being expanded?